spec/system/management/proposals_spec.rb

```diff
@@ -47,6 +47,15 @@
 
       expect(page).to have_content "User is not verified"
     end
+
+    scenario "when user has not been selected we can not create proposal" do
```

Comment:
```suggestion
    scenario "when user has not been selected we can not create proposals" do
```

Or:

```suggestion
    scenario "when user has not been selected we can not create a proposal" do
```

Choose the same options as in budgets :wink:.